### DISCUSSION TITLE
🔮 Locales

### DISCUSSION THREAD

**Discussion by im-parsa** [im-parsa/elite-website#4] [Q&A] [answered]
👀 Make sure to tell us:

- Are you following or changing anything in Locale schema
What branch/version you are using
- Are you changing or creating a new locale
- Are you adding any new method that doesn't exist on [Locale Schemas (Reply or Error)]

Also, don't forget to [Join our Discord Community](https://discord.gg/RUrks4JqW6)

**Comment by IntelligentQuantum** [comment, marked as answer]
My idea is to hire several people for translation so that we can make the community bigger and more famous.
